spanner/spannertest/db.go

```diff
@@ -632,8 +635,9 @@ func (t *table) addColumn(cd spansql.ColumnDef, newTable bool) *status.Status {
 	}
 
 	t.cols = append(t.cols, colInfo{
-		Name: cd.Name,
-		Type: cd.Type,
+		Name:    cd.Name,
+		Type:    cd.Type,
+		NotNull: cd.NotNull,
```

Comment:
down on line 718 there's a TODO that could be at least partially done, namely setting this value in an `ALTER TABLE`. No need to do any validation right now (leave that as a TODO), but may as well maintain the colInfo.NotNull field.

Reply:
Certainly, that was worth doing now.

Done, apart from the validation based on the requirement to be not null.
https://github.com/googleapis/google-cloud-go/pull/3491/commits/840de7cdb3e5954dd3d1d4f3845014281de1e9f0

Btw, I updated the below related codes in this func as well because the previous code didn't seem to accept a simple statement that just adds/removes Not Null without doing any alternation.